index.js

```diff
@@ -0,0 +1,87 @@
+module.exports = {
+  parser: "@typescript-eslint/parser",
+
+  extends: ["plugin:react/recommended", "prettier/@typescript-eslint", "plugin:prettier/recommended", "prettier/react"],
+
+  plugins: ["simple-import-sort", "jest", "@typescript-eslint", "react-hooks", "react", "prettier", "import"],
+
+  ignorePatterns: ["coverage", "**/dist", ".cache-loader", "node_modules", ".yalc"],
+
+  parserOptions: {
+    ecmaVersion: 2018,
+    sourceType: "module",
+    ecmaFeatures: {
+      jsx: true,
+    },
+  },
+
+  settings: {
+    react: {
+      version: "detect",
+    },
+  },
+
+  env: {
+    browser: true,
+    node: true,
+    jest: true,
+    es6: true,
+  },
+
+  rules: {
+    "prettier/prettier": "error",
+
+    // Base
+
+    "no-unused-expressions": ["error", { allowTernary: true }],
+    "no-console": "off",
+    "no-confusing-arrow": "off",
+    "no-else-return": "off",
+    "no-return-assign": ["error", "except-parens"],
+    "no-underscore-dangle": "off",
+    camelcase: "off",
+    "prefer-arrow-callback": ["error", { allowNamedFunctions: true }],
+    "class-methods-use-this": "off",
+    "no-restricted-syntax": "off",
+    "no-param-reassign": ["error", { props: false }],
+    "arrow-body-style": "off",
+
+    // Imports
+
+    "simple-import-sort/sort": "error",
+    "sort-imports": "off",
+    "import/order": "off",
+    "import/no-extraneous-dependencies": "off",
+    "import/first": "error",
+    "import/newline-after-import": "error",
+    "import/no-duplicates": "error",
+
+    // Typescript
+
+    "@typescript-eslint/no-unused-vars": [
```

Comment:
Could we use an override for this rule and apply it only to tests?
I feel like this is the only place we may want to have unused vars

Reply:
Yeah that makes sense - or can ignore the line on a case by case basis in tests instead.

We have unused variable detection disabled in `tsconfig` already, so making it error here is a good idea. Reason to disable in `tsconfig` is because when it's enabled iterative development becomes a pain in the ass because typescript literally won't re-build until they're all gone.